resources/sdk/clisdkclient/scripts/preprocess-swagger.js

```diff
@@ -58,8 +58,18 @@ const inclusionList = {
 		}
 	},
 	"/api/v2/routing/queues/{queueId}/users": {
```

Comment:
gc queues now has an additional "user" command with the following methods: get, join, remove